.gitignore

```diff
@@ -2,3 +2,21 @@ Data
 Data_Net
 .ipynb_checkpoints
 __pycache__
+
+
+# Environments
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+
+PTX/Experiments/ARUNet-MiddleLines-Pipeline/data/
```

Comment:
I favor directory specific ignores to go into a .ignore for that directory.    That is, move these to the PTX/... subdir

Reply:
Agreed.

Reply:
Makes sense.  Will do